src/env/test/mod.rs

```diff
@@ -27,6 +29,8 @@ impl core::fmt::Write for TestWrite {
     }
 }
 
+pub type TestCustomization = CustomizationImpl;
```

Comment:
Let's inline this. We don't need to hide the actual type since we don't access it with just traits.

Reply:
done